### ISSUE TITLE
Error: Method 'RunTargetAsync' does not have an implementation.

### ISSUE THREAD

**Issue by jnm2** [agc93/Cake.BuildSystems.Module#6]
On Cake 0.23, and Cake.BuildSystems.Module 0.1.5, I am now getting this output:

```cs
Error: Method 'RunTargetAsync' in type 'Cake.TravisCI.Module.TravisCIEngine' from assembly 'Cake.TravisCI.Module, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.
```

I have no issues with this on TFS, only when running via command line on my dev machine where Travis is not in the picture.

**Comment by eddiegroves** [comment]
Looks like this has a fix, any reason it hasn't been closed and a new release published to nuget?

**Comment by jnm2** [comment]
@agc93 published this fix as https://www.nuget.org/packages/Cake.BuildSystems.Module/0.2.0-unstable0009 because he wanted to do more testing, but it seems to be at least as good as 0.1.5.

**Comment by agc93** [comment]
And in fact a fully updated version tested against Cake 0.24.0 has now been published as [0.2.0 on NuGet](https://www.nuget.org/packages/Cake.BuildSystems.Module/0.2.0)
